Runtime/RestrictiveEnvironment/DefaultStepLockHandling.cs

```diff
@@ -11,12 +11,33 @@ namespace Innoactive.Creator.Core.RestrictiveEnvironment
     /// </summary>
     public class DefaultStepLockHandling : StepLockHandlingStrategy
     {
+        /// <summary>
+        /// If you want to disable lock handling set this parameter to false.
+        /// </summary>
+        public const string EnableLockHandlingParameterName = "EnableLockHandling";
+
+        /// <summary>
+        /// To stop locking every LockableProperty at the beginning of the training set this parameter to false.
+        /// </summary>
+        public const string LockCourseOnStartParameterName = "LockCourseOnStart";
+
+        /// <summary>
+        /// To lock every LockableProperty at the end of the training set this parameter to true.
+        /// </summary>
+        public const string LockOnCourseFinishedParameterName = "LockOnCourseFinished";
```

Comment:
Inconsisted naming:

1. It should be LockOnCourse...
2. It should be either LockOnCourseStart and LockOnCourseFinish, or LockOnCourseStarted nad LockOnCourseFinished.

Personally, I would prefer LockWhenCourseStarts and LockWhenCourseFinishes